wiki/Beatmapping/Flow/en.md

```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - linear flow
+  - oval flow
+  - circular flow
+---
+
+# Flow
+
+The **flow** is a mapping concept that describes natural and intuitive movement of a player throughout the map. The flow, or lack thereof, is almost always related to the [osu!](/wiki/Game_mode/osu!) game mode beatmaps due to the two-dimensional playfield. Additionally, the osu! beatmaps offer a vast variety of [patterns](/wiki/Beatmap/Pattern) and slider shapes, which may be used by mappers to direct a player's cursor wherever they want to.
+
+If a mapper deliberately ignores the concepts of flow, or uses their opposites, they utilise [antiflow](/wiki/Beatmapping/Antiflow).
+
+## Key components
+
+While flow is mostly subjective, as movement preferences differ from one person to another, there are a few ideas that give the feeling and impression of flow on a majority of patterns or whole beatmaps.
```

Comment:
i feel like there should be more emphasis on the subjectivity of this somehow, since lots of mappers think flow is an outdated term that should never be used nowadays lol